src/FSharpVSPowerTools.Core/Utils.fs

```diff
@@ -420,7 +420,32 @@ module String =
         elif value.Contains(pattern) then
             Some value
         else None
-     
+    
+    open System.IO
+
+    let getLines (str: string) =
+        use reader = new StringReader(str)
+        [|
+        let line = ref (reader.ReadLine())
+        while (!line |> function | null -> false | _ -> true) do
```

Comment:
Bummer. http://latkin.org/blog/2015/05/18/null-checking-considerations-in-f-its-harder-than-you-think/

I guess I complained because using `function` is a bit weird here. Perhaps `match !line with` is more natural?